diff --git a/learning/arithmetics.c b/learning/arithmetics.c
index 610f278..6c90bae 100644
--- a/learning/arithmetics.c
+++ b/learning/arithmetics.c
@@ -1,9 +1,11 @@
 #include <stdio.h>
 
+// Funcion de suma
 int add(int sumA, int sumB){
 	return sumA+ sumB;
 }
 
+//Funcion de resta:
 int rest(int delA, int delB){
 	return delA+ delB;
 }
diff --git a/learning/arithmetics.h b/learning/arithmetics.h
index 2a82062..70d9f7b 100644
--- a/learning/arithmetics.h
+++ b/learning/arithmetics.h
@@ -1,7 +1,13 @@
 #ifndef ARITHMETICS_H
 #define ARITHMETICS_H
+// Global variables
 int variableA;
 int variableB;
+// Static variables
+
+//Constant variables
+ 
+// Funcitons
 
 int add(int sumA, int sumB);
 
diff --git a/learning/main.c b/learning/main.c
index 9d36202..da4ae81 100644
--- a/learning/main.c
+++ b/learning/main.c
@@ -19,6 +19,10 @@ int main(){
 	
 	printf ("La suma de los 2 numeros es: %d + %d = %d\n",variableA, variableB, add( variableA, variableB));
 	printf("La division de los 2 numeros es: %d + %d = %f\n",variableA, variableB, divide( variableA, variableB));
+	
+	/*
+	Agregando comentarios al proyecto
+		*/
 	return 0;
 }
 
